ldk/node/src/hostClients/networkClient.test.ts

```diff
@@ -58,7 +58,7 @@ describe('NetworkClient', () => {
 
     beforeEach(async () => {
       sentResponse = new Messages.HTTPResponseMsg()
-        .setData('response data')
+        .setData(new Uint8Array([173, 235, 41, 162, 123, 30, 117, 171, 90]))
```

Comment:
It's not clear how this value is derived - maybe we should start with text, decode into uint8 explicitly?

Reply:
What if I add an encoder and just encode the message? With the changes below, it seems that it would always return as a uint8 array